### DISCUSSION TITLE
[UNSTABLE] Boarding pod disappears if multiple players interact with it

### DISCUSSION THREAD

**Discussion by DownloadableContent5021** [FakeFishGames/Barotrauma#14708] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

If have a crewmember inside the new boarding pod in a depth charge loader and another crewmember attempts to get inside while the pod is still inside the loader, then the first crewmate will be kicked out and the boarding pod will disappear.

### Reproduction steps

1. Open submarine editor with Barsuk (or any ship that has a depth charge loader).
2. Place boarding pod inside depth charge loader using editor, then hit play.
3. have one crewmate enter boarding pod, switch to another crewmate, then while boarding pod is inside the depth loader, attempt to enter the pod as well using "E".
4. watch as the first crewmate is kicked out and the boarding pod disappears.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

Other

### -

v1.6.9.0 (unstable)

### Which operating system did you encounter this bug on?

MacOS

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly. Fix coming in the next Unstable build. Closing ticket.
